WordPress/Classes/Services/PageCoordinator.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+class PageCoordinator {
+    typealias TemplateSelectionCompletion = (String?) -> Void
```

Comment:
The string return value here is in preparation for adding the template slug or contents depending on what path works out the best then.